### DISCUSSION TITLE
Disable FPS coutner/meter (top right corner)?

### DISCUSSION THREAD

**Discussion by jkellermeier** [BepInEx/BepInEx#730] [Help and Q&A] [answered]
Disable FPS coutner/meter (top right corner)?

Hey,

I tried to find a solution via google - but "Shift-U" was all I found but didn't worked.
Is there a shortcut or config option to disable the FPS counter/meter from BepInEx?

Kind regards
Julian

**Comment by toebeann** [comment, marked as answer]
BepInEx doesn't have an FPS counter. You might have a mod _for_ BepInEx installed which is adding an FPS counter, though. Where did you get BepInEx from?

**Reply by jkellermeier** [reply]
Got my answer - thanks!
Seems that the counter came from Better Networking or Serverside Simulation.
